release/pkg/operations/upload.go

```diff
@@ -88,11 +94,7 @@ func UploadArtifacts(r *releasetypes.ReleaseConfig, eksArtifacts map[string][]re
 					// Trim -helm on the packages helm chart, but don't need to trim tinkerbell chart since the AssetName is the same as the repoName
 					trimmedAsset := strings.TrimSuffix(artifact.Image.AssetName, "-helm")
 
-					helmDriver, err := helm.NewHelm()
-					if err != nil {
-						return fmt.Errorf("creating helm client: %v", err)
-					}
-
+					fmt.Printf("Modifying helm chart for %s\n", trimmedAsset)
```

Comment:
No HelmLog here?

Reply:
It's in a different dir, so didn't want to import it in.